src/sendMessage.js

```diff
@@ -7,10 +7,15 @@ var log = require("npmlog");
 module.exports = function(mergeWithDefaults, api, ctx) {
   return function sendMessage(msg, thread_id, callback) {
     if(!callback) callback = function() {};
-    if(typeof msg !== "string") return callback({error: "Message should be of type string and not " + typeof msg + "."});
+    if(typeof msg !== "string" && typeof msg !== "object")
```

Comment:
`typeof array` will also be object. The proper way to check for object would be
```js
Object.prototype.toString.call(msg) === '[object Object]'
```

I'll push a fix for the other places where we did this.